enterprise/admin/__init__.py

```diff
@@ -607,6 +616,9 @@ def has_delete_permission(self, request, obj=None):
         """
         Disable deletion for EnterpriseCourseEnrollment.
         """
+        features = getattr(settings, 'FEATURES', {})
+        if features.get(constants.ALLOW_ADMIN_ENTERPRISE_COURSE_ENROLLMENT_DELETION, False):
+            return True
```

Comment:
Idk if I'm completely missing something obvious but couldn't we just do `return features.get(constants.ALLOW_ADMIN_ENTERPRISE_COURSE_ENROLLMENT_DELETION, False)`

Reply:
hah, yes :)